girder/api/v1/user.py

```diff
@@ -42,7 +42,8 @@ def __init__(self):
         self.route('GET', ('me',), self.getMe)
         self.route('GET', ('authentication',), self.login)
         self.route('GET', (':id',), self.getUser)
-        self.route('GET', (':id', 'details'), self.getUserDetails)
+        self.route('GET', (':id', 'count'), self.getUserDetails)
```

Comment:
Actually what happened was I think you changed the old endpoint on accident, rather than your new one below.